scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -225,5 +232,47 @@ task ExtractTask {
      }
  }
 
+task GetLastModifiedDateOrUniqueValue {
+    # because this is being used to determine if the data has changed, never use call cache
+    meta {
+        volatile: true
+    }
+
+    input {
+        String data_project
+        String default_dataset
+        String dataset_table
+        File? service_account_json
+    }
+
+    String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+
+    # ------------------------------------------------
+    # try to get the last modified date for the table in question; if something comes back from BigQuwey
+    # that isn't in the right format (e.g. an error), throw it away and generate something unique
+    command {
+        if [ ~{has_service_account_file} = 'true' ]; then
+            gcloud auth activate-service-account --key-file='~{service_account_json}'
+        fi
+
+        LASTMODIFIED=$(bq show --location=US --format=json --project_id=~{data_project} ~{default_dataset}.~{dataset_table} | python3 -c "import sys, json; print(json.load(sys.stdin)['lastModifiedTime']);")
```

Comment:
LFMF: you may want location to be a parameter.  We recently had an issue where we've added regionality and US was hardcoded everywhere and the BQ apis failed.

Reply:
Also, if this gets called a lot, is python overkill?  Would it be faster to just use bash (though I get that python is way easier to read)

Reply:
Wait, sorry, I misread this.  I'm not sure what is standard in? Is that how we pass information into the runners?

Reply:
The `bq show` command is piped (becomes `stdin`) to the python which parses the JSON to grab the `lastModififedTime` field.

Reply:
Oh derp, you're right nevermind